### ISSUE TITLE
Cannot build firebase_snippets_app for Android

### ISSUE THREAD

**Issue by jlara310** [firebase/snippets-flutter#26]
When I try to  build the `firebase_snippets_app` locally, I get the following errors:

```
                             ^
  symbol:   method setAppCacheEnabled(boolean)
  location: class WebSettings
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':flutter_webview_plugin:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.
```

```
WARNING: This version of flutter_webview_plugin will break your Android build if it or its dependencies aren't compatible with AndroidX.
         See https://goo.gl/CP92wY for more information on the problem and how to fix it.
         This warning prints for all Android build failures. The real root cause of the error may be unrelated.
```

**Comment by jlara310** [comment]
Things I tried:

* Upgrading flutter_webview_plugin to 0.4.0. Same error.
* Tried using Android studio to migrate as describe in https://docs.flutter.dev/release/breaking-changes/androidx-migration, but did not work.
* Tried https://docs.flutter.dev/release/breaking-changes/androidx-migration#what-if-i-cant-use-android-studio. Created a new project and move the flutter dart files over. After this, I was able to tweak things a bit and run the Firestore snippets.
